OpenGL/Math/Matrix2.cs

```diff
@@ -86,6 +86,45 @@ public static Matrix2 Identity
             }
         }
 
+        public float this[int column, int row]
+        {
+            get
+            {
+                if (row == 0)
+                {
+                    if (column == 0) return row1.X;
```

Comment:
Once you have determined the row, why not just use `row1[column]`?  That would eliminate all of the `column` checks in this code.